docs/source/channel_update_tutorial.rst

```diff
@@ -579,7 +579,7 @@ The first step is to package the Basic chaincode:
 
 .. code:: bash
 
-    peer lifecycle chaincode package basic.tar.gz --path ../asset-transfer-basic/chaincode-go/ --lang golang --label basic_1
+    peer lifecycle chaincode package basic.tar.gz --path ../asset-transfer-basic/chaincode-go/ --lang golang --label basic_1.0.1
```

Comment:
Apologies for not reviewing sooner, I think I did not understand the intent of why use v1.0.1 instead of v1. Could you explain? Does test-network default to v1.0.1?

Reply:
@denyeart

>Apologies for not reviewing sooner, I think I did not understand the intent of why use v1.0.1 instead of v1. Could you explain? Does test-network default to v1.0.1?

No problem at all, please don’t worry about it. Thank you for your review.

The latest test-network defaults to using version 1.0.1, as indicated in [this code](https://github.com/hyperledger/fabric-samples/blob/3826626d0049258abe256379be6a69bf4c687fee/test-network/network.config#L29C13-L29C18). My intention was to align with this update.

While addressing your comment, I noticed a discrepancy regarding the default chaincode version in the latest test-network (e.g., 1.0.1, [1.0.0](https://github.com/hyperledger/fabric-samples/blob/3826626d0049258abe256379be6a69bf4c687fee/test-network/scripts/packageCC.sh#L23), [1.0](https://github.com/hyperledger/fabric-samples/blob/3826626d0049258abe256379be6a69bf4c687fee/test-network/scripts/utils.sh#L75)), including an inconsistency in the command message description.

Would it be more appropriate to resolve this discrepancy by unifying the default value to '1' in the fabric-samples repository, rather than addressing it through this PR? I would appreciate your thoughts on the best approach.

Reply:
I think the value of `CC_VERSION="1.0.1"` was a mistake. Prior to this [commit](https://github.com/hyperledger/fabric-samples/commit/e73bb717db47185a8ccd1b701503b369cd2a73e1) it was `"1.0"`

And in the lifecycle ReadTheDoc tutorials it is "1.0".

I think "1.0" was chosen for samples to differentiate it from the sequence number which is an integer like "1".

I'd suggest to align the samples default and the tutorials to be "1.0".

Thank you for looking into these details!

Reply:
@denyeart 
Thank you for your response.

I agree with your suggestion. To align with the tutorials, I will create and submit a patch to fix the version to “1.0” in the fabric-samples repository. Once that's done, I will update this PR accordingly.

Thank you again for your guidance!

Reply:
@denyeart 
I updated the PR to align to "1.0".
Also, I submitted the following PR in fabric-samples. 
https://github.com/hyperledger/fabric-samples/pull/1245

Please confirm them.